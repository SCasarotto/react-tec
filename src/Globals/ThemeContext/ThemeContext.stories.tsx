import React from 'react';

import { Story, Meta } from '@storybook/react';

import { ThemeProvider, ThemeProviderProps } from './ThemeContext';

export default {
  title: 'Globals/ThemeContext',
  component: ThemeProvider,
  argTypes: {},
} as Meta;

const Template: Story<ThemeProviderProps> = (args) => (
  <ThemeProvider {...args} />
);

export const Default = Template.bind({});
Default.args = {};
