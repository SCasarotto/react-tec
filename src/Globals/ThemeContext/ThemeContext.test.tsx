import React from 'react';

import { render } from '@testing-library/react';

import { ThemeProvider } from './ThemeContext';

describe('ThemeContext', () => {
  // Renders Without Error
  it('matches snapshot', () => {
    const { asFragment } = render(<ThemeProvider />);
    expect(asFragment()).toMatchSnapshot();
  });
});
