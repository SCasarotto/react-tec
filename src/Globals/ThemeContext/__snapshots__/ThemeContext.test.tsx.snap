// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeContext matches snapshot 1`] = `<DocumentFragment />`;
